app/controllers/admin/dashboard_controller.rb

```diff
@@ -1,6 +1,14 @@
 class Admin::DashboardController < ApplicationController
 
     def show
-    end 
+      @admin = current_user
+      @users = User.all
+      @orders = Order.all
+      @packaged_orders = Order.packaged_orders
+      @pending_orders = Order.pending_orders
+      @shipped_orders = Order.shipped_orders
+      @cancelled_orders = Order.cancelled_orders
```

Comment:
I dont think creating all these variables was the best way but its the way I got it to work for now.